### ISSUE TITLE
Integrating updates

### ISSUE THREAD

**Issue by SSJ9899TDylan** [Skulltrail192/One-Core-API-Binaries#11]
Back when I integrated your updates into nLite, only the App-Compat installer worked. However, I actually couldn’t get the Setup to work!

**Comment by Adahel** [comment]
Nlite is a bug. Use the SVCPACK installation:
Edit the file: AMD64 \ SVCPACK.INF
If the file does not exist open with the winrar: AMD64 \ SVCPACK.IN_ and extract the file. Delete SVCPACK.IN_. In the file SVCPACK.INI [SetupHotfixesToRun] on the next line write
"[name] .exe / q / n / z". Place the file in the \ AMD64 \ svcpack folder. Please someone explain to him, I'm not good at explanations. Sorry if I wrote it wrong. I do not know English.

**Comment by Skulltrail192** [comment]
Maybe error occours because i use sfxcab, instead official sfx from Microsoft. If somebody know how use sfx from Microsoft, please tell me

**Comment by Skulltrail192** [comment]
Hello Adahel, do you can make it for us?

**Comment by Adahel** [comment]
My video card broke. I'm following git on my smartphone. I'll do it as soon as my computer is working. While my computer was good I noticed something different in the command line integration (update. exe/?). I may need to replace a DLL from another package, but I can only confirm it after testing. I'm just talking because I'm a committed person. As I have not found anywhere: I can pass to you the link (not by git) of my ISO Windows XP x64 with all updates. I made it myself.

**Comment by SSJ9899TDylan** [comment]
That would be awesome.

**Comment by SSJ9899TDylan** [comment]
Can you send the ISO to me when it’s ready?

**Comment by Adahel** [comment]
Yes, but not for the git. I do not want to lose my ISO for copyright. Warning: You must have a copy of Windows XP Professional x64, Windows XP Professional x86 is a different edition of version x64. Architecture: AMD64.

**Comment by Adahel** [comment]
Warning: Addon SVCPACK is not a true addon. SVCPACK is processed at T-13 installation sector. The addon I'm going to do is SVCPACK.

**Comment by SSJ9899TDylan** [comment]
Ok

**Comment by Adahel** [comment]
I just finished the addon x64: packages from 2 months ago. On the tests appeared repeatedly an error message. I can not distribute in the company I work for if there are errors, even if they do not interfere with operating system operation. When I run Windows Update, it shows updates already installed. Not to anything wrong with the sfx. Screenshot and download link:
<a href="https://ibb.co/n50O2y"><img src="https://thumb.ibb.co/n50O2y/Update.png" alt="Update" border="0"></a> <a href="https://ibb.co/dywKaJ"><img src="https://thumb.ibb.co/dywKaJ/Windows_XP_one_core_api.png" alt="Windows_XP_one_core_api" border="0"></a> <a href="https://ibb.co/c5TAhy"><img src="https://thumb.ibb.co/c5TAhy/XPBUG.png" alt="XPBUG" border="0"></a>
https://1drv.ms/f/s!AtzOKJLp3od6lrIaQtYBwkZHpeAm0Q
ISO x64 soon. To do x86 is just replace the sfx. See the files is not too difficult. It is possible to correct the problem of the error message? Sorry for being late. I'm still going to do x86, when I have iso x86 to test.

**Comment by SSJ9899TDylan** [comment]
Do the x86 true addons.
Requirements for addons:
Windows XP uSP4 ISO or Post-SP3 AIO Update Pack
Windows XP POSReady True Addon (optional, but recommended).
I’ll update it even higher to the latest commits afterwards.
Can those addons work with RVM Integrator?

**Comment by SSJ9899TDylan** [comment]
I’ll test later

**Comment by SSJ9899TDylan** [comment]
I also might need a Windows Server 2003 Post-SP2 AIO Update Pack for testing
on Windows Server 2003 & for R2 too

**Comment by SSJ9899TDylan** [comment]
Are these true addons compatible with Server 2003?

**Comment by Adahel** [comment]
I only remember one true addon incompatible with Server 2003. I did not warn you, but the x86 is ready for some time.
